README.md

```diff
@@ -1,2 +1,53 @@
-# hostal_las_paisas
+<h1 align="center"> hostal_las_paisas </h1>
+
 Basic hostel page with quoting tool and subscription support.
+
+## Read 📄
+
+* [Mobile Mockups](https://scene.zeplin.io/project/5ca139b23af358054bcab181)
+* [Website Style Guide and Mockups](https://scene.zeplin.io/project/5ca139b23af358054bcab181)
+* [Downloadable resources](https://app.zeplin.io/project/5ca139b23af358054bcab181/dashboard)
+* [Website Prototype](https://sketch.cloud/s/0xOEG/R1Zx1o8/play)
+* [Website zepllin](https://sketch.cloud/s/0xOEG)
+* [CSS style guide](https://github.com/airbnb/css#css)
+* [BEM - CSS guide](http://getbem.com/introduction/)
+* [JS style guide](https://github.com/airbnb/javascript)
+
+## Instructions 🚀
+
+look at the **Develop** branch to learn how to deploy the project.
+Then open git bash and write the following code as it is: git clone https://github.com/rtsc8/hostal_donde_las_paisas.git
+then execute the index.html, it will open in your default browser
+
+### Pre-requisites 📋
+
+Have a web browser (preferably Google Chrome)
+
+## Deployment 📦
+
+We can find an introductory Hero to the page, also a carrousel with a diversity of images and a footer with references for contact
```

Comment:
we don't have a deployment strategy jet, so this section content can be just a "TODO"